src/components/event/details/DetailsAddons.vue

```diff
@@ -92,6 +92,14 @@ export default Vue.extend({
     renderAddonValue(key: string | object, value: any): string {
       if (key === 'window') {
         return value.innerWidth + 'x' + value.innerHeight;
+      } else if (key === 'userAgent' && typeof value != 'string') {
+        let formattedUserAgent = `${value.browser} ${value.browserVersion} / ${value.os} ${value.osVersion}`;
+
+        if (!value.browser) {
```

Comment:
you are using this variable above. That means that you expect that this is defined

Reply:
There is a chance that user-agent data will not be parsed in the catcher and then the `value.browser` will be empty. I can change it to strict equality with an empty string

Reply:
if so, this statement should be above the previous